js/igv-create.js

```diff
@@ -326,7 +326,7 @@ var igv = (function (igv) {
             browser.$searchInput = $('<input type="text" placeholder="Locus Search">');
 
             browser.$searchInput.change(function (e) {
-                browser.parseSearchInput( $(e.target).val() );
+                browser.parseSearchInput( $(e.target).val().toLowerCase() );
```

Comment:
This part is to to make the input case insensitive while changing. And it is working fine.

Please also implement this when the page first get loaded. For example, when I set 
locus: "Chr7:55,085,725-55,276,031" in the copyNumber.html, it is going to return 
"Unrecognized locus Chr7:55,085,725-55,276,031"